### ISSUE TITLE
PGH004 rule requires colon in `noqa`

### ISSUE THREAD

**Issue by jurasofish** [charliermarsh/ruff#3872]
If you run `ruff check scratch_171.py --isolated --select PGH004` on the snippet below you get the output

```
scratch_171.py:2:8: PGH004 Use specific rule codes when using noqa
```

```python
a = 1  # noqa: blahblahblah
a = 1  # noqa blahblahblah
```

PGH004 is incorrectly detecting that `noqa blahblahblah` does not have a specific rule code, presumably because it does not have a colon.

```
ruff --version
ruff 0.0.260
```

**Comment by charliermarsh** [comment]
Hm, I think this is arguably correct. Both Ruff and Flake8 treat `# noqa blahblahblah` as equivalent to `# noqa`. Even `# noqa E501` is treated as identical to `# noqa` (that is, it's treated as a blanket ignore). So it seems right to be called out as such?

**Comment by jurasofish** [comment]
Ah you're right - I had assumed that `noqa blah` was equivalent to `noqa: blah` but it's actually a blanket ignore.

[PGH004](https://beta.ruff.rs/docs/rules/#pygrep-hooks-pgh) plus [RUF100](https://beta.ruff.rs/docs/rules/#ruff-specific-rules-ruf) protect against that pretty well then.

thank you
